chain/ethereum/src/capabilities.rs

```diff
@@ -73,7 +73,11 @@ impl_slog_value!(NodeCapabilities, "{}");
 impl graph::blockchain::NodeCapabilities<crate::Chain> for NodeCapabilities {
     fn from_data_sources(data_sources: &[DataSource]) -> Self {
         NodeCapabilities {
-            archive: data_sources.iter().any(|ds| ds.mapping.requires_archive()),
+            archive: data_sources.iter().any(|ds| {
+                ds.mapping
+                    .requires_archive()
+                    .expect("failed to parse mappings")
```

Comment:
Is this a breaking change? Where does this error go to?

Reply:
This panicking behaviour already happened deep down the `requires_archive` method, we're just bubbling up the error handling here.